paths/migrate.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package paths
```

Comment:
you need a blank line before this, otherwise the copyright comment is the package doc

you can verify with `go doc .` in the directory

Reply:
otherwise LGTM with the caveat that I haven't actually patched it in and tried it. But seems right and I trust your testing.

Reply:
Added blank line.